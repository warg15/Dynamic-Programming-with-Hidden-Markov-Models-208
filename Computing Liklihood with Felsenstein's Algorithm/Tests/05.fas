>Adelie-penguin
CGTGGTTTCGTTACTACGAATTGCACCGTTATATGGACGAACTCTGTATTCCTTATAAAAGCCGTATCTATCGATTCCGCCGTCTGACTACTCGTGGACGAGATGAAAAGACCGTCCCTCTATTCGCACCAGAAGAAGGGTTGGAGAGAGGAAAACGAAGCGTGGCCGCGAACTTATGGTAAAGATTTTTCCGTGGGATT
>Red-throated-loon
CGTGGTTTCCTTACTAAGAATTGCACGGTTATATGGACCAGCTTTGTATCCCTGACAAAAGCCGTATCTGTCGATTCCGCCGTCTAACTGCTCGTGGACGAGATGAAAAAACCATACCTTTATTCGCACCAGAAGAAGGGTTGGAAAGAGGAAAACGCAGCGTGGCCGCGTACTTATGGTAAAGGTTCTTCCGTGGGATT
>Annas-hummingbird
CGTGTTTTCGTTACTAAGAGTTGGACGGTTACATGGACCGGCTTCGTATTCCTTACAAAAGCCGTATCTGTCGATTCCGCCGACTAACTAGTAGTTGACGAGATGAAAAGACCATCTCTCTATTCGTACCAGAAGAAGGGTTAGTAAGAGAAAAACGAAACGTGGCGGCGAACTTGCGGTAAAGATCCTTGCGTGGGATT
>Pigeon
CGTGGTTCCGTTACTAGGAACTGCACAGTTGTATGGACCAGCCTTGTATCCCTTACAAAAGCCGTATCTGTCGAGTCCTCCGTCTAACTACTCGTCGACGAGAGGATAAGACCATCCCTCTATTCGCACCAGAGGAAGGTTCGGAAAGAGGAAAACGAAGCTTAGCTGCTAACTTGTGGTAAAGATTCTTCTATAGGATT
>Medium-ground-finch
TGTGGTTTCGTTACTTAGACTTGCGCGGTAATACGAACCAATTTTGTATCGCGCAACAAGGCCTTGCCTGTTGATTCCGCCGTATAACTACTCGTGGATGAGATGAAAAGATCATCCTCCTTATCGGACCAGAAGAAGGGTTGGAAAAAGAAAAACGAAGCGTGGCCGCGCACTTATGGTAGAGATTCTTCCTTGGAATT
>Chuck-wills-widow
CGTGGCTTCGTTACTAAGAACTGCACGGTTATATAGACCAGCTTCATATTCCTTACAAAAGCCATATCTGTCAATTCCGTCGTCTAACTACTCGTGGACGAGGTGGAAAGACCATCCCCATATTCGCACCAGAAGAAGGGTTGGAAATAGGAAAACGAAATGCGGCCGCGAACTTATGGTAAAGATTCTTGCGTGGGGTT
>Peregrine-falcon
CGTGGTATGGTTACTAAGAACTGCACGGTTATATGGACCAGCTTTGTATTCCTTACAACAGCCGTATCTGTCGATTCCGCCGTCTAACTACTCGTGGGCGAGATGAAAAGACCATCCTTCTAGTCGCACCAGAAGAAGGATTGGAAAGAAGAAAACGAGGCGTGGTCGCGAACTTATGGTAAGGACTCTGCCGTGGGATC
>Pekin-duck
CGTGGTTGCGTTGGTAAGAATTGTACGGTTATATGGAACAGCTTTGTATTCCTTACAACAGCCGTATCTGTTGATTCCCCCGTGCAACTACTGGTGGGCAACATGTAATGACCATCCCTGTATGGGCACCAGAAGAAAGGCTGGAAAGAGGAAAACGAGGCATGGCCGCGAACTCGTGGTAAAGAGTCTTCCAAGGGATC
>American-crow
TGTGGTTCCGTTACTAAGACTTGCACGGTAATACGAACCAGCTTTGTATCGCGTAACAAAGCCGCGTCTGTTGATTCCGCCGTATAAATACGCGTGGACGAGATAATAAGACCATCCCCCTTATCGCACCAGTAGAAGGGTTGGAAAAAGGAAAACGAAACGTGGCCGCGCACTCATGGTAGAGATTCTTCCGTGGGATT
>Bald-eagle
CGTGGTTTCGTTACTAAGAATTGCCCGATTATATGGACCAGCTTTGTATTCCTTACAAAAGCCGTATCTGTCGATTCCGCCGTCTAACTACTCGTGGACGAGATGAGAAAACCATCCCTCTATTCGCACTAGAAGAAGGGTTGGAAAGAGGAAAACGAAGCGTGGTCGCGAACCTATGGTAAAGATTCTTCCGAGGGATT
>Cuckoo-roller
CGTGATTTCGTTACTAAGAATTGTACGGTTATACGGGCCAGTTTTGTATTCCTAACAAAAGCCGTATCTGTCGATTCCGCCGTCTAACTACTCATCGACGAGATGAAAAGACCATCCCTCTCTTCGCACGATAAGAAGGGTTGGAAAGAGGAAAACGATGCGTGGCCGCCGACTTATGGTTAAGAGTCTTTCGTGGGATT
>MacQueens-bustard
CGTGGCTTCGTTACTAAGAATTGCACGGTTATATGGACCAGCTTTGGATTCCTGACAAAAGCCGTATCTGTCGATTACGCCGTCTAACTACCCGTGGACGAGATGAAAAGACCATCCCTCTATTCGCACCAGAAGGAGGGATGGAAAGAGGAAAACGAAGCGTGGCCGCGGACTTATGGTAATGACTCTTCCGTGGGATT
>Killdeer
CGTGGTTTCGTTACTAAGAATTGTACGGTTGTATGGACCAGCTATGTATTCCCTACAAAAGTCGTATCTATCGGTTGCGCCGTCTGACTACTCGTGGACGTGATCAAAAGACCATCTCTCTATTCGCACCAGAAGAAGGGTTGGAAAGAGGAAAACGAAGCGTGGCCGCGAACTTATGGTACAGATTCTTCCGTGGGATT
>White-throated-tinamou
CACAGTTCCGGTACTGAGAACTGTATGGTTATATCAACCATCCTTGTATTCCTTACAACAGCCGTATCTGCCGATTCCGCCGTCTAACTACGCGCAGAAGAAGGGAAATGACCATCCATTCATTCGCGCCGGAAGAAGGGTTCGGAAGAGTCAAACGAAGCGCTGCGGCGAACTTGTGTTAAAAAGTCTTCCATCGGATC
>Downy-woodpecker
CGCTGTATCGTTACTAAGAATTGTACGCTTATATCAAGCAACTTTGTATGCCTTACAGAAGTCGTCTCTGTCGATTCCGCTGAATATCTCCTCGTGAACAAGATGAGAAGACCTTTCCTCGATTCGCATCACAAGAAGGGTTGGAAAGAGAAAAGCGAAGCGTGACCACAAACTTATGATAAAGACTTTTCCGTCGGATT
>Turkey-vulture
CGTGGTTTCGTTACTAAGAATTGCACGGTCATATGGACCAGCTTTGTATTCCTTACAAAAGCCATATCTGTCGATTCCGCCATCTAACTACTCGTGGACGAGATAAGAAGACCATCCCTCTATTCGCACTAGAAGAAGGGTTGGAAAGAGGAAAACGAAGCGTGGCCACGAACTTATGGTAAAGATTCTTCCGTGGGATT
>Common-cuckoo
TGTGTTTTCGTTACTAAGAACTGCACGACTATATGGACCAGCTTTGTATTCCTTACAAAAGCCCTATTTGTCGATTCCGCCGTCTCACTGCTCCTGGACAAGATAAATAGACCACCCCTCTATTCGCACCAGAAGAAGGGTTGGAAAGAGAAAAACGAAGCGTGGCCGCGGACTTATGGTAAAGATTCTTCCGTGAGATT
>Great-crested-grebe
CGTCGTTTTGTCACCAAGAATTGCACGGCTGTACGGACCGTCTTTATATTCCTTACAAAAGCCGTATCTGTCGATTCCGCCGTCTAACTACTCGTGGACAAGATGAAAAGACCATCCCTCTATTCGCACCAGAAGACTGGTTGGAATGAGGAAAACGAAGCGTGGCCGCGAACTTATCGTAAAAATTCTCTCATGGAATC
>Carmine-bee-eater
CGTGGTTTCGTCACTAAAAATTCCGCGGTTGTATGGACCAGTTTTGTATTCCTTACAAAAGCCGTATCTGTCGATTCCGATGTCTAACTACTCGTGGACGAGATGATAAGACTATCCCTCTATCCACACCGGAAGAAGGATTGGAAAGAGGAAAACGAAGCGCGGCCACGAACTTATGGTAAAGATTCTTCCGTGGGATT
>Barn-owl
CGTGGTTTCGTTACTAAGAATTGGACGGTTATATGGACCAGCTCTGTATTCCTTATAAAAGCCGTATCTGTCGATTCCGCCGTCTAACTACTCGTGGACGAGATGAAAAGACCATCCTTCTATTCGCACCAGAAGAAGAGTTGGAAAGAGGAAAACGAAGCGTGGCCGCGAACTTATAGTAAAGACTCTTCCGTGGGATT
>Kea
CGTGGTTTCGTTTCTAAGAATTGCACGACTATATGGATCAGCTTTGTATTCCTTAATAAAGCCGTATCTGCCGATTCCGCCGTTTGGCTACTCGTAGACGAGATGAAAAGACCGTCCCTTTATTCGCACCAGAAGAAGGGTTGGAAAGAGGAAAACGAAGCGTGGCCGCGAACTTATGGTAAAGATTCTTCCGTGGGACT
>Brown-mesite
CGTGGATTCGTTACGAAGAATTGCACGGTTGTATGGACCAGCTTTGTATTCCGTACGAAAGCCGTATCTATCGATTCCGCTGTCTAGCTACTCGGGGACGAGATGAAAAGACCATCCCTCTATTCGCACCAGAAGAAGGGTTGGAACGAGGAAAACGAAGCGTGGCTGCGAGCACATGGTCACGATTCTTCCATGGGAAT
>Red-legged-seriema
CGTGATTTTGTTACTAAGAATTGCACGGCTATATCGACCAGCTTTATATTCCTTACAAAAGCCGTATCTGTCGCTTCCGCTGTCTAACTACTCGTGGCCGAGATGAAAGGACCATCCCTCTATTCGCACCAGAAGAAGGGTTGAAAAGAGGAAAACGAGGCGTGGCAACGAACTTAGCGTAAAGATTCTTCCGTGGGAAT
>Common-ostrich
CACAGTTCCGGTACTAAGAACTGCACGGTTATATGGAGCAACTTTGTATTCCTTACAACAGTCGTATCTGTTGATTCCGCCGTCTAACTACTCGTGGACGAAGGGAAATGACCATCCCTCCATTCGCGCCAGAAGAGGTGTTGGAAAGAGTCAAACGAAGTGTTTTGGCGAACTTGTGGTAAAGAGTCTTCCAGTGGATC
>Dalmatian-pelican
CGTGGTTTCGTTACTAAGAATTGCACCGTTATATGAACGAGCTTTGTATTCCTTATAAAAGCCGTATCTGCCGATTCCGCCGTCTAGCTACTCGTGGACGAGTTGAAAAGACCATCCCTCTATTCGCACCAGAAGAAGGGTTCGAAAGGGGAAAACGAAGCGTGGCCGCGAACTTATGGTAAAGTTTCTTCCGTGGGATT
>Emperor-penguin
CGTGGTTTCGTTACTAAGAATTGCACCGTTATATGGACGAACTTTGTATTCCTTATAAAAGCCGTATCTATCGATTCCGCCGTCTGACTACTCGTGGACGAGATGAAAAGACCGTCCCTCTATTCGCACCAGAAGAAGGGTTGGAAAGAGGAAAAGGAAGCGTGGCCGCGAACTTATGGTAAAGATTTTTCCGTGGGATT
>Sunbittern
CGTAGTCTCTTTACTAAGAATTGCACGGTTATATGGATCAGTTTTGTATTTCTTATAAAAGCCGTATCTGTGGATTCCGCCGTCTAACTACTCGTGGACGAGATGAAAAAGCCATCCCTCAGTTCACACCAGAGTAAGGGTTGTAAGGAGGAAAACGAAGTGTGGCAGCGAACTAATAGAAAAAATTCTGCCGTGGGATT
>Northern-fulmar
CGTGGTTTTGTTACTAAGAATTGCACCGTTATATGAACGAGCTTTGTGTTCCGTGTAAAAGCCGTATCTGTCGATTCCGCCGTCTAACTACTCGTGGACGAGATGAAAAGACCATCCCTCTATTCGCACCAGAAGAAGGGTTGGAAAGAGGAAAACGAAGCGTGGCCACGAACTTATGGTAAAGATTCTTGCGCGGGATT
>Great-cormorant
GGTGGTTTCGTTTCTAACAATCGCACCGTTATATGGACGAGCTTTGTATTTCTTACAAAAGCCGTATCTGTCGATTCAGCCGTCTAACCACTCGTGGACGAGATGAAAAGACCATCCTTCTATTCGCACCAGAGGAAGGATTGGAAAGAGGAAAACGAAGCGTGGCCGCGAACTCATGGTAAAGTTTCTTCCGTGTGAGT
>Bar-tailed-trogon
CGTGGTTTCGTTACTAAGAATTGCACGGTTATATGGACCAGCTTTGTAATCCTTACAAAGGCCGTATCTGTCAATTCCGCTGGGTAACTACTTGTGGACGATATGAAGCGACCACCCCTCTATTCGCACCAAAAGATGAGTTGAAAAGGGGAAAACGAAGCGTGGCCGCGAACTTTTGGTATAGTTTCTTCCGTGGGATT
>Little-egret
CATGGTTTCGTTACTAAGAATTGCACCGTTATATGGACGAGCTTTGTATTCCTTATAAAAGCCGTATCTGTCGATTCCGCCGTCTAACTACTCGTGGACGAGATGAAAAGACCATCCCTCCATTCGCACCAGAAGAACGGTTCGAAAGAGGAAAACGAAGCGTGGCCGCAAACTTATGGTAAAGTCGCATCCGTGGAATT
>Rifleman
CGTGGTTTCGGTACTAAGACTTGCATGGGAATATGAACCAGCTTTGTATCGCTTAACAAAATCGTTGCTGTTGATTCCGCCGTCTAACTACCCGTGGACGATAAGAAACGACCATCCCTCTATTCGCAACAGAAGAAGGGTTGGAAAAAGAAAAACGAAGCGTGGCCGCGCACTTATGGTAAAGATGCTTCCGTGGGATT
>Red-crested-turaco
CCTAGTTTCGTTACTACGAATTGCACGGTCATAAGGACCAGTTTTGAATTCGTTACAGAAACCGCATCTGTCGATTCCGCCGTCTAACTACTCGTGTACGAGATGAAAAGACCATCCCTCTATTCGCACCAAAAGAAGGGTTGGAAAGAGGAAAACGAAGCGTGGTCGCGGACTTATGCTAAAGATTCCTCCGTGGGATT
>Hoatzin
CGTGGTTTCGTGACTAAGCATTACACACTTGTATGGGCCAGCTTTGTATTCCTTACAAAAGCCGTATCTGTCGATCACGCCGTCTAACTACTCGTAGACGAGATGAAAAGACCATCCCCTTATTCGCACCAGAAGAAGGGTTGGAAAGAGGAAGATGAAGCGTGGCCGCGAACTTAAAGTAAAGATTCTTCCGTGGGATC
>Chimney-swift
CGTGGTTTCGTTACTAAGAATTGCACGCTTACATGGACCAGCTTTGTCTTCCTTCTTAAAATCGGGTCCGTCGATCCCACCGTCTAACTACTCGTTCAGGCGATAAAAAGACCATCTATCTGTTCGCACCAGAAGGGGGGTTGGAAAGAGGAAAAAGGAATGTGGCCGCGAACTTATGGTAAAGATTCTTGCATTGGACT
>Chicken
CGAGGTTTCGGTACTAAGAATAGTACGGTCGTATGGACCAGGTTTGTATTCCTTAGAACAGCCGTAGCTCTTGATTTCGCCGTGTAATTACTGGTGAACGAAACGCACTGACCATCCCTCTATCGGCACCAGAAAAAGGGTTGGATAGAGGAAAATGAGGCGTGGCCGCCAACTTATGGTAAAAAGTCTGCCAAAGAATC
>Rhinoceros-hornbill
TGTGGTTTCGTTACGAACAATTGCATGGTTACATGGACCAGCTTTGTATTCCTTACAAAAACCGTATCTCTCGATTCCGCCGTCTAATGACTCGTGAATGAGGTAAAAAGACCATGCCTCTACTCGCATCAGAAGGAGTGTTAGAGAGAGCAAAACGAAGCGTGGCCACGAACTTATGGCAAAGATTCTTCCATGGGATT
>White-tailed-tropicbird
CGTGGTTTCATTACTAAGAATTGCACGGTTATATGGACCAGGTTTGTATTCCTTAAAAAAGCCGTATCTGTCGATTCTGCCGTCCAACTTCTCGTGGACGAGATGAAAAGACCATCCCTTTATTCGCACCAGAGTAAGGGTGGGAAAGAGGAAGACGAAGCGTGGCCGCAAACTTATGGTAAAGGTTCTTCCGTGGGATT
>Turkey
CGAGGTTTCGGTACTAAGAATGGTACGGTCGTATGCACTAGGTTTGTAATCCTTAGAACAACCGTATCTCTTGATTTCGCCGTGTAATTACTGGTGAACGAAATGCAATGACCATCCCTCTATCGGCGCCAGAAAAAGGGCTGGATAGAGGAAAATGAGGCGTGGCCGCGAACTTATGATAAAAAGTCTGCCAAAGAATC
>Crested-ibis
CGTGGTTTCGTTACTAAGAATTGCACCGTTATATGGACGAGCTTTGTATTCCTTACAAAAGCCGTATCTGTCGATTCCGCCGTCTAACTACGCGTGGACGAGATTAAAAGACCATCCCTCTATTCGCACCAGAAGAAGGGTTTGAAAGAGGAAAACTAAGCGTGGCCGCGAACTTATAGTAAAGTTTCTTCCGTGGGATT
>Zebra-finch
TGTGGTTTCGTTACTTAGACTTGCGCGATAATACGAACCAGCTTTGTATCGCGTAACGAGGCCGTGCCTGTCGATCCCGCCGTATAACTACTCGTGGATGAGACGAAAAGACCATCCCCCTTATCGGACCGGAAGGAGGGTTGGATAAAGAAAAACAAAGCGCGGCCGCGCACTTATGGTAGAGATCCTTCCTTGGAATT
>Yellow-throated-sandgrouse
CGTGGTTTCGTTACTAAGAATTGCACGGTTATATGGACCAGCTTTGTATTCCTTACAAAAGCCGTATCTGTCCACTCCGCCGTCTAACTACTCATGGACGAGATGAAAAAACCATCACTCTTTTCGCACCAGAGGAAGGGTAGGAAAGAGGAAAACGAAGCGTGGCCGCAAATTTATGGTAAAGATTCTTGCATAGGATT
>White-tailed-eagle
CGTGGTTTCGTTACTAAGAATTGCCCGATTATATGGACCAGCTTTGTATTCCTTACAAAAGCCGTATCTGTCGATTCCGCCGTCTAACTACTCGTGGACGAGATGAGAAAACCATCCCTCTATTCGCACTAGAAGAAGGGTTGGAAAGAGGAAAACGAAGCGTGGTCGCGAACCTATGGTAAAGATTCTTCCGAGGGATT
>Speckled-mousebird
CGTGATTTCGTTACTAAGAATTGCACGGTTATATGGGCCAGCTTTGTATTTCTTACAAAAGTCGTATCTGTCGATTCAGCCGTCTAACTACTCATGGACCAGATGAGAAGACCATCCCTCTATTCGCACCAGAAGAATGGTTGGAAAGAGGAAAACGAAGTGTGGCCGCGAATCTATGGTGAAGATTCTTCCGTGGGATT
>Budgerigar
CCTGGTTTCGTCACTAAGAATTGCACGGTTATATGGATCAGCTTTGTATTCCTTAAAAAAGCCGTATCTGGCGATTCCGACGTCTAACTACTCGTGGACGTTCTGAAAAGACCATCCCTCTATTTGCACAAGAAGAAGGGTTGGAAAGAGGAAAACGAAGCGTGGCCGCGAACTTATGGTAAATATTTTTCCGTGGGACT
>Grey-crowned-crane
CGTGGTTTCACTACTAAGAATTGCAAGGTTGTATGGACCTGCTTTGTATTCCTTACAAAAGCCGTATCTGTCGATTCCGCCGTCTAACTACTCGTGGACGAGATGAAAAGACCATCCCTCTATTCGCACCAGAAGAAGAGTCGTAAAGAGGAAAACGAAGCGTGGCCGGGAACTTATGGTAAAGATCCTTCCGTGGGATT
>Golden-collared-manakin
CGTGGTTTCGTTACTAAGACTTGCACGGTAATATGGACCAGCTTAGTATCGCGTAACAAAGCCGTGTCTGATAATTCCGCCGTTTAACTACTCGTGGGCGAGATGTAAAGATCACCCCCCTATAAGCACCAGAAGAAGGGTTGGAAAAAGAAGCACGAAGCGTGGCCGCGCACTTATGGTAAGGATTCTTCCGTGGCATT
>American-flamingo
CGTGGTTTTGTTACTAAGAATTGCACGGCTATATGGACCATCTTTGTATTCCTTACAAAAGCCGTATCTGTCGATTCCGCCGTCTAACGACTCGTAGACGCGATGAAAAGACCATCCCTCTATTCGCACCAGAAGAAGGGTTGGAAAGAGGAAAACGAAGCGTGGCCGCGAACTTATCGTAAAGATTCTTCCATGGGATT
