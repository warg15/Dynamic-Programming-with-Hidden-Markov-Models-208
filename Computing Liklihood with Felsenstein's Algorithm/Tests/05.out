-3566.3734323348954
